src/styles/miscStyles.js

```diff
@@ -200,8 +198,7 @@ export default ({ color, backgroundColor, borderColor, cardColor, dividerColor }
     flex: 1,
     flexDirection: 'row',
     height: CONTROL_SIZE,
-    marginTop: 5,
-    marginBottom: 5,
+    marginVertical: 5,
```

Comment:
This one should likely be `HALF_SPACING` as at some point we used `5` and `10` for the margins.

Reply:
Change made.